examples/cfd/06_poisson.ipynb

```diff
@@ -278,17 +281,17 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "CPU times: user 1 µs, sys: 1e+03 ns, total: 2 µs\n",
-      "Wall time: 4.05 µs\n"
+      "CPU times: user 169 ms, sys: 9.68 ms, total: 178 ms\n",
+      "Wall time: 179 ms\n"
```

Comment:
I changed the magic command for this cell from %time (time this command) to %%time (time this cell).
The pseudo time operator is now twice as slow as the buffer stepping scheme.

Reply:
This is now fixed. As pointed out by @FabioLuporini the last cell included the compilation of the operator as well as it running. I have moved the time command to only run on the operator now.